app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -49,6 +83,36 @@ angular.module('miq.util').factory('chartsMixin', function() {
       units    : __('KBps'),
       dataName : __('KBps'),
       tooltipFn  : dailyTimeTooltip
+    },
+    dailyPodUsageConfig: {
+      chartId     : 'podUsageDailyChart',
+      headTitle   : __('Pod Creation and Deletion Trends'),
+      subTitle    : __('Last 30 Days'),
+      createdLabel: __('Created'),
+      deletedLabel: __('Deleted'),
+      tooltip     : {
+        contents: dailyPodTimeTooltip,
+        position: lineChartTooltipPositionFactory('podUsageDailyChart'),
+      },
+      point       : {r: 1},
+      size        : {height: 130},
+      color       : {pattern: [pfUtils.colorPalette.blue, pfUtils.colorPalette.green]},
+      grid        : {y: {show: false}},
+      setAreaChart: true
+    },
+    dailyImageUsageConfig: {
+      chartId     : 'imageUsageDailyChart',
+      headTitle   : __('New Image Usage Trends'),
```

Comment:
@simon3z :+1: done

p.s
We have _Network Utilization Trends_ , I am not an English speaker so I just copied. If it's true to image trend it should be true to network ?